### PR TITLE
feature/rebrand

### PR DESCRIPTION
**Pull request by brett-sargeant**
rebranding for Department of Infrastructure, Regional Development and Cities change of ownership.

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -45,10 +45,8 @@ const Header = props => (
         <div className={style.subtitle}>Framework Dashboard</div>
       </div>
       <div className={style.orgContainer}>
-        <a className={style.orgLink} href={LINKS.ORG}>
```
Comment (jonalport) [inline src/components/Header/Header.js:48]: Also remove the associated styles, constants and assets, it they are no longer used?

File: src/components/FooterContainer/Footer/Footer.js [hunk 2]
```diff
@@ -11,14 +11,9 @@ import {
   EMAIL_ADDRESSES,
 } from '../../../constants';
 import Link from '../../Link/Link';
-import logo from '../../../images/australia-dta-logo.svg';
-import dtaLogo from './dtaLogo.svg';
-
+{ /* import logo from '../../../images/crest1.png';  */ }
```
Comment (jonalport) [inline src/components/FooterContainer/Footer/Footer.js:14]: Rather delete than comment out, as the code can always be found again in the version history. Also delete associated resources if no longer required.

**Review by jonalport — APPROVED** [review]
Looks good, just a couple comments. I think the repo is currently set to require a review before merging. This can be turned off.

**Comment by brett-sargeant** [comment]
@jonalport are you able to modify my permissions to let me merge?

**Pull request merged**